### PR TITLE
feat(email): add utm params to the post verify email

### LINKED ISSUE #110: Add campaigns to post-verification links
So that we better can track post-verification email success, please add the following campaigns to the link:

Android Link:
https://www.mozilla.org/en-US/firefox/android/?utm_source=email&utm_medium=email&utm_campaign=fx-account-verified

iOS Link:
https://www.mozilla.org/en-US/firefox/ios/?utm_source=email&utm_medium=email&utm_campaign=fx-account-verified

Connect another Device button and text link alternative:
https://www.mozilla.org/en-US/firefox/sync/?utm_source=email&utm_medium=email&utm_campaign=fx-account-verified

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #110

### CODE REVIEW

**Comment by vladikoff** [comment]
@vbudhram @rfk r?

**Comment by vladikoff** [comment]
image

Text-only:

```
Account verified!

You successfully connected your first device to your Firefox Account. Now you can sign in to Sync from your other devices, including Firefox for Android and iOS.

Sign in to Sync: https://www.mozilla.org/en-US/firefox/sync/?utm_source=email&utm_medium=email&utm_campaign=fx-account-verified

This is an automated email; if you received it in error, no action is required. For more information, please visit https://support.mozilla.org/kb/im-having-problems-with-my-firefox-account
```

File: mailer.js [hunk 1]
```diff
@@ -280,7 +280,10 @@ module.exports = function (log) {
   Mailer.prototype.postVerifyEmail = function (message) {
     log.trace({ op: 'mailer.postVerifyEmail', email: message.email, uid: message.uid })
 
-    var link = this.syncUrl
+    // special utm params, just for this email
+    // details at github.com/mozilla/fxa-auth-mailer/issues/110
+    var postVerifyUtmParams = '?utm_source=email&utm_medium=email&utm_campaign=fx-account-verified'
```
Comment (vbudhram) [inline mailer.js]: Should this be configurable?

Comment (vladikoff) [inline mailer.js]: Probably not, it is the same in all environments, nothing to configure IMHO

**Comment by vbudhram** [comment]
:+1:

**Comment by vladikoff** [comment]
@rfk r? merge if looks good

**Comment by rfk** [comment]
r+, let's ship it this train

File: mailer.js [hunk 2]
```diff
@@ -292,8 +295,8 @@ module.exports = function (log) {
       template: 'postVerifyEmail',
       templateValues: {
         link: link,
-        androidUrl: this.androidUrl,
-        iosUrl: this.iosUrl,
+        androidUrl: this.androidUrl + postVerifyUtmParams,
+        iosUrl: this.iosUrl + postVerifyUtmParams,
```
Comment (rfk) [inline mailer.js]: This'll break if we ever configure one of the URLs to have existing query parameters, may be better off using a URL joining library.  But we can do that in a follow-up.

**Pull request merged**